lms/static/scripts/frontend_apps/components/test/SvgIcon-test.js

```diff
@@ -1,94 +1,97 @@
 import { createElement, render } from 'preact';
```

Comment:
This file was copied from `src/shared/components/svg-icon.js` in the client repo with only changes to file path references.